tests/test_main.py

```diff
@@ -4,7 +4,6 @@
 
 from app.main import Car, CarWashStation
 
-
 def test_car():
```

Comment:
Do not make changes in test files

Reply:
Although the change is minor, but remember not to touch any test file, README or other config files that were present before you cloned the repository

Reply:
Everything else looks ok